pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -718,8 +716,15 @@ def cmd_connection_save(context, name):
     connections = context.connections_repo
 
     context.spinner_stop()
-
-    connections.add(save_connection.name, save_connection)
+    try:
+        connections.add(save_connection)
+    except Exception as exc:
```

Comment:
The broad exception handling can now be narrowed to catch only `(ConnectionsFileError, IOError)` (or just `ConnectionsFileError` if the `IOError` gets handled in `add()` as well, see my comment on that).